src/client/components/app-core/text.css

```diff
@@ -1,3 +1,20 @@
 html {
-  font-family: sans-serif;
+  font-family: 'system', sans-serif;
 }
+
+a {
```

Comment:
These styles apply to all links everywhere in the site. Also in a paragraph and wherever. So I think you want to keep these a bit more neutral, specifically not `text-transform: uppercase;`, the rest may be neutral enough. And safe that uppercase for links in menus (different ticket on trello).